.github/workflows/checks.yml

```diff
@@ -36,6 +36,10 @@ jobs:
     name: "Unit Tests"
     runs-on: ubuntu-latest
 
+    env:
+      TF_APPEND_USER_AGENT: TEST
+      OPENTOFU_USER_AGENT: TEST
```

Comment:
I think this user agent might be a bit too nondescriptive if this is the entire user agent?

Reply:
the full user agent is OPENTOFU_USER_AGENT+TF_APPEND_USER_AGENT. We can change the APPEND_USER_AGENT env var depending on where the test is being run from. This is just a placeholder for now.

Reply:
In that case the user agent should be a bit more verbose, such as `OpenTofu-integration-test`

Reply:
Done